obs-browser-source.cpp

```diff
@@ -87,6 +87,18 @@ BrowserSource::BrowserSource(obs_data_t *, obs_source_t *source_)
 				   obs_module_text("RefreshNoCache"),
 				   refreshFunction, (void *)this);
 
+	auto jsEventFunction = [](void *p, calldata_t *calldata) {
+		DispatchJSEvent(calldata_string(calldata, "eventName"),
```

Comment:
`calldata_string()` can return NULL, and `DispatchJSEvent` takes an `std::string`, so it would be nice to have some basic validation for these fields. Consider a situation where no JSON string is defined, if an event does not require data.

Reply:
Added NULL checks for both eventName and jsonString